### DISCUSSION TITLE
Is it likely the Cotton Bureau t-shirts will return?

### DISCUSSION THREAD

**Discussion by oliverdunk** [parcel-bundler/parcel#5079] [General] [answered]
I'd love one of the t-shirts or hoodies [listed on Cotton Bureau](https://cottonbureau.com/products/parcel-t-shirt), and referenced on the website.

Are the sales of these something you have control over, and are they likely to become available again?

I notice the logo is different to what's in use now, perhaps that needs updating.

**Comment by devongovett** [comment, marked as answer]
I will contact them and see if we can get something going. 😄

**Reply by devongovett** [reply]
Update, they are now available on demand!
